docs/current/sdk/nodejs/snippets/github-google-cloud/main.mjs

```diff
@@ -39,7 +39,7 @@ connect(async (daggerClient) => {
     template: {
       containers: [
         {
-          image: `${gcr.publish}`,
+          image: gcrContainerPublishResponse,
```

Comment:
@vikram-dagger : I'm not sure about this one

Reply:
Should we also change it in line 59?

Reply:
Oh yes you're right, I forgot about this one:
`gcrContainerPublishResponse.publish`